templates/artist/includes/artist_detail_overview.html

```diff
@@ -50,7 +50,7 @@ <h4 class="investing">${{ campaign.amount_raised|intcomma }}</h4>
                     {% if campaign.percentage_funded == 100 %}
                         <h3 class="campaign-funded">FUNDED!</h3>
                     {% endif %}
-                    <p><em>Campaign ended {{ campaign.end_datetime|date:'F jS' }}</em></p>
+                        <p><em>Campaign ended {{ campaign.end_datetime|date:'F jS' }}</em></p>
```

Comment:
No, you got this backwards. Unindent the line you changed here, indent the `{% endif %}` line below.